### PR TITLE
Removing certain unneded cluster invalidation messages

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Removing certain unneded cluster invalidation messages

### CODE REVIEW

**Comment by agnivade** [comment]
This contains my changes in https://github.com/mattermost/mattermost-server/pull/13537 too. Would it be better to exclude those changes in this PR, so that both of them can be merged independantly ?Otherwise, my PR becomes redundant :stuck_out_tongue_closed_eyes:

**Comment by jespino** [comment]
@agnivade done

File: app/cluster_handlers.go [hunk 1]
```diff
@@ -13,8 +13,6 @@ func (a *App) RegisterAllClusterMessageHandlers() {
 	a.Cluster.RegisterClusterMessageHandler(model.CLUSTER_EVENT_PUBLISH, a.ClusterPublishHandler)
 	a.Cluster.RegisterClusterMessageHandler(model.CLUSTER_EVENT_UPDATE_STATUS, a.ClusterUpdateStatusHandler)
 	a.Cluster.RegisterClusterMessageHandler(model.CLUSTER_EVENT_INVALIDATE_ALL_CACHES, a.ClusterInvalidateAllCachesHandler)
-	a.Cluster.RegisterClusterMessageHandler(model.CLUSTER_EVENT_INVALIDATE_CACHE_FOR_WEBHOOK, a.ClusterInvalidateCacheForWebhookHandler)
-	a.Cluster.RegisterClusterMessageHandler(model.CLUSTER_EVENT_INVALIDATE_CACHE_FOR_CHANNEL_POSTS, a.ClusterInvalidateCacheForChannelPostsHandler)
```
[the remark this answers is not in the document]
Comment (jespino) [reply app/cluster_handlers.go]: that constants are used in the metrics code in the enterprise repo, and is part of the `model` module, which means is part of our public api, I prefer to not remove it yet, and make the cleanup as part of the 6.0 version. I going to create a ticket for that.

**Comment by jespino** [comment]
@agnivade I added a simplified version of the doc string that you propose, because I don't want to leave a comment that can be obsolete soon, I prefer to leave it as is expected to be in the future (and is valid already).

File: app/cluster_handlers.go [hunk 2]
```diff
@@ -9,12 +9,15 @@ import (
 	"github.com/mattermost/mattermost-server/v5/model"
 )
 
+// RegisterAllClusterMessageHandlers register the cluster message handlers that are handled by the App layer
```
Comment (agnivade) [inline app/cluster_handlers.go]: - _registers_ the ... 
- Add a period to the end of the sentence.

**Comment by agnivade** [comment]
@ogi-m - Would you test this post-merge ? Please let us know. We would like to get this pushed soon as it is one of the causes of performance degradation affecting our cluster.

**Comment by jespino** [comment]
@ogi-m there is not a ticket, but is the same problem here https://mattermost.atlassian.net/browse/MM-21396 but related to webhooks cache and for channel last post time cache. I'll duplicated the ticket and link it here.

**Comment by jespino** [comment]
@ogi-m here you have: https://mattermost.atlassian.net/browse/MM-21612

File: app/cluster_handlers.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jespino) [reply app/cluster_handlers.go]: I created the ticket here: https://mattermost.atlassian.net/browse/MM-21473

**Pull request merged**